src/core/shapes.coffee

```diff
@@ -132,6 +132,9 @@ defineShape 'Image',
       img = new Image()
       img.src = data.imageSrc
     createShape('Image', {x: data.x, y: data.y, image: img, scale: data.scale})
+  move: ( moveInfo={} ) ->
+    @x = @x - moveInfo.xDiff
+    @y = @y - moveInfo.yDiff
```

Comment:
I think I would prefer an absolute `setUpperLeft()` instead of delta-based `move()`. As we make more use of it, the logic will be simpler for the caller.